content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+
+- **outputs.tf** - contains the output variables to expose information.
+
+- **provider.tf** - contains the provider configuration and version.
+
+<br /> <br />
+
+## Deploy Terraform
+After updating the **inputs.tf** file, and carefully reviewing the other files, initialize the Terraform provider:
+<br />
+
+```bash
+terraform init
+```
+
+The `init` command above will download plugins and providers from the **provider.tf** file. Next, preview the resources Terraform will create. 
+<br />
+
+```bash
+terraform plan
+```
+
+The output displays the resources Terraform will create in an actual implementation. Lastly, finish creating all resources using the following command. 
+<br />
+
+```bash
+terraform apply -auto-approve
+```
+
+It will take up to 20 mins to provision a cluster. See a sample successful deplyment in the visual below. 
+While the deployment is in progress, you can switch to the Palette to validate the in-progress deployment. 
+
+<br /> <br />
+
+
+## Check the In-Progress Deployment
+Log back into the [Palette console](https://console.spectrocloud.com/), and navigate to the **Profile** section in the left **Main Menu**. If the terraform deployment is successful, you will see the newly created Profile, as shown in the visual below. 
+
+<br /> <br />
+
+![Visual of the successful Profile in the Palette console.](/tutorials/create-pack/registries-and-packs_create-pack_verify-profile.png)
+
+<br /> <br />
+
+## Validate
+
+Navigate to the **Clusters** section in the Palette console's left **Main Menu** and view the cluster deployment in progress. After the Palette finishes provisioning the cluster, the status will turn to **Running** and **Healthy**. You can access the application on the exposed port 8080 for the **Hello Universe** service. 
+
+<br /> <br />
+
+![Visual of the cluster health.](/tutorials/create-pack/registries-and-packs_create-pack_cluster-health.png)
+
+<br /> <br />
+
+The application will look similar to the visual below. 
+
+<br /> <br />
+
+![Visual of the successful accessing the application using the load balancer URL.](/tutorials/create-pack/registries-and-packs_create-pack_success.png)
+
+<br /> <br />
+
+Congratulations, you've successfully deployed the *Hello Universe* application to the cluster. 
+
+<br /> <br />
+
+## Cleanup
+
+To delete the resources, switch back to the "tutorials" container, and run the following command from within the **terraform/pack-tf** directory:
+<br />
+
+```bash
+terraform destroy -auto-approve
+```
+<br /> 
+
+Wait for the resources to clean up and ensure the successful resource deletion in the Palette console as well. 
```

Comment:
We need to mention the force delete scenario and how to terminate the tutorial container.  

![CleanShot 2023-04-21 at 17 55 44](https://user-images.githubusercontent.com/29551334/233753253-d5f8b532-f83d-4190-9a10-c0f428ab15fe.png)